### DISCUSSION TITLE
Inav X8 how to re-set the mixer

### DISCUSSION THREAD

**Discussion by jockypower** [iNavFlight/inav#9178] [Q&A] [answered]
I use Matek H743 v3 FC, and two FCHUB 12S. And from 8 output to connect 8 seperated ESC. 

In Inav configaration 6. if I collect 8X, no mix guider can change the moter order. If I collect X4, it have the mix guider button. I donot know why. 

Now, How I can change the mixer. Any one can help to answer. thanks.

**Comment by DzikuVx** [comment, marked as answer]
You can use this tool to build any mixer you want https://www.iforce2d.net/mixercalc/
